content/webinars/getting-started-policy-as-code-2020-04-09/index.md

```diff
@@ -0,0 +1,72 @@
+---
+# Name of the webinar.
+title: "Getting Started with Policy as Code for Any Cloud"
```

Comment:
Don't hold up this PR, but more generally I am curious what the guidance is around using "Policy as Code" vs. "CrossGuard" when talking about PaC.

Reply:
@isaac-pulumi ☝️